cheroot/workers/threadpool.py

```diff
@@ -111,11 +111,6 @@ def run(self):
                 if conn is _SHUTDOWNREQUEST:
                     return
 
-                # Just close the connection and move on.
-                if conn.closeable:
```

Comment:
this is not strictly needed, but there's no reason to send the conn through the queue to be closed, when we can close it immediately within `connections`